ArtOfIllusion/src/artofillusion/PluginRegistry.java

```diff
@@ -52,7 +52,7 @@ public class PluginRegistry
   public static Map<String, Throwable> notifyPlugins(int message, Object... args)
   {
     Map<String, Throwable> errors = new HashMap<>();
-    categoryClasses.getOrDefault(Plugin.class, List.of()).forEach(plugin -> {
+    categoryClasses.getOrDefault(Plugin.class, Collections.emptyList()).forEach(plugin -> {
```

Comment:
Please keep refactors like this, as well as comment re-formats &tc. in a separate commit from your functional changes.